admin/client/src/components/Tabs/README.md

```diff
@@ -1 +1,37 @@
 # Tabs
+
+For separating content into tabs without the need for separate pages.
+
+This extends from `react-bootstrap` with similar expected behaviours, only difference is that when
+there is only 1 tab (or none) in the Tabset only the content will show without the clickable Tab.
```

Comment:
Grammar police: Numbers from one through nine should be spelled out (http://www.grammarbook.com/numbers/numbers.asp). Definitely `1` = `one`. Also, inconsistent capitalisation of `Tab`